### PR TITLE
[LLM Runtime]Fix gptneox bug

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
fix bug
API changed or not
not 
## Description

detail description 
JIRA ticket: None

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by zhentaoyu — APPROVED** [review]
Thanks. My bad.

File: intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp [hunk 1]
```diff
@@ -45,6 +45,7 @@ void model_load_internal(const std::string& fname, model_archs arch, model_conte
   std::unique_ptr<GPTNEOX> ms(new GPTNEOX());
   ms->init(fname.c_str(), lctx, n_gpu_layers, use_mmap, use_mlock, vocab_only);
   ms->load(lctx, progress_callback, progress_callback_user_data);
+  lctx.support_jblas_kv = true;
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp:48]: when can we set it as default true? @DDEle

**Review by airMeng — APPROVED** [review]

File: intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp (hunk 1 above)
Comment (DDEle) [reply intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp:48]: Most of models now support fused-attention now: https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/docs/fused_attention.md#supported-models

IMHO, it is better to let model owners to explicitly confirms that they have implemented fused-attention support for their models.

Comment (airMeng) [reply intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp:48]: @a32543254 @zhentaoyu @zhenwei-intel @CeciliaWwq  any comments?

Comment (zhentaoyu) [reply intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp:48]: OK for me

**Pull request merged**